pkg/chains/signing/kms/kms.go

```diff
@@ -41,6 +43,19 @@ type Signer struct {
 // NewSigner returns a configured Signer
 func NewSigner(ctx context.Context, cfg config.KMSSigner) (*Signer, error) {
 	kmsOpts := []signature.RPCOption{}
+
+	// Checks if the vault address provide by the user is a valid address or not
+	if cfg.Auth.Address != "" {
+		vaultAddress := strings.TrimPrefix(cfg.Auth.Address, "http://")
+		vaultAddress = strings.TrimPrefix(vaultAddress, "https://")
```

Comment:
do you think using `url.Parse()` for this would be more reliable? https://pkg.go.dev/net/url#Parse

Reply:
yeah that make sense 👍🏻